app/Enums/RclResponsesEnum.php

```diff
@@ -11,7 +11,8 @@ enum RclResponsesEnum: string
     case Cancelled = "RCL/CLA REJECTED\nCLEARANCE CALLED REVERT TO VOICE PROCEDURES";
     case Invalid = "RCL REJECTED\nINVAILD %s\nRESUBMIT YOUR REQUEST";
     case Contact = "RCL/CLA RECEIVED\nCONTACT %s BY VOICE";
-    case Acknowledge = "RCL/CLA RECEIVED\nFLY CURRENT FLIGHT PLAN OR AS AMENDED BY ATC\nCONTINUE TO MONITOR NATTRAK UNTIL ENTERING OCA";
+    case Acknowledge = "RCL/CLA RECEIVED\nCONTINUE TO MONITOR NATTRAK UNTIL ENTERING OCA";
+    case AcknowledgeMoved = "RCL/CLA RECEIVED BY %s\nCONTINUE TO MONITOR NATTRAK";
```

Comment:
defer to others on whether this is the correct verbiage

Reply:
https://github.com/vatsimnetwork/nattrak/pull/142/commits/b0d865a4c8147748458d9e79d73580e2bdd75e44